assets/javascripts/discourse/connectors/topic-navigation/no-answer.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if show}}
-  {{#topic-navigation-popup}}
+  {{#topic-navigation-popup noticeId="solved_notice"}}
```

Comment:
I'm not sure if this variable needs to be `snake_case` for some reason, but if we can make it `solved-noticed` that matches our front end code better :)